### DISCUSSION TITLE
由于服务故障导致的正版登录问题（综合）

### DISCUSSION THREAD

**Discussion by lactobionicAcid** [Meloong-Git/PCL#4147] [讨论]
> [!IMPORTANT]
> **请勿将该问题作为 Issue 反馈给龙猫。**
该问题与 PCL 启动器无关，同时龙猫也不具有修复该问题的能力。

> [!NOTE]
> 本问题已解决。

10 月 30 日 0 时左右（北京时间），Microsoft Azure Front Door 服务发生故障，致使大部分与 Microsoft 相关的服务不可用，包括 Minecraft 正版登录和认证。

Azure 官方已彻底修复该故障。

现在，与正版登录相关的所有服务均已恢复正常。

[查看 Azure 状态](https://azure.status.microsoft/zh-cn/status)  |  [查看正版登录失败率](https://eed575d5.clsshare.com/s)(链接失效)  |  [查看 Xbox 状态](https://support.xbox.com/zh-CN/xbox-live-status)

<img width="567" height="201" alt="image" src="https://github.com/user-attachments/assets/919f0097-070f-4317-8edd-7a2418dec64f" />

**Comment by Chiloven945** [comment]
https://bugs.mojang.com/browse/WEB-7272

---

补充下图片
![image](https://github.com/Hex-Dragon/PCL2/assets/116699482/9a84ec51-026e-4fa2-904f-385c801e06c6)

**Comment by allMagicNB** [comment]
Xbox Live Status：
https://support.xbox.com/zh-CN/xbox-live-status
